### DISCUSSION TITLE
Create fetch with default options?

### DISCUSSION THREAD

**Discussion by Matman94** [nuxt/nuxt#16563] [Questions]
In the [documentation](https://github.com/unjs/ohmyfetch#%EF%B8%8F-create-fetch-with-default-options) is written like this:

```
✔️ Create fetch with default options
This utility is useful if you need to use common options across serveral fetch calls.

Note: Defaults will be cloned at one level and inherrited. Be careful about nested options like headers.

const apiFetch = $fetch.create({ baseURL: '/api' })

apiFetch('/test') // Same as $fetch('/test', { baseURL: '/api' })

```
Cool, but in what file should I put it? In what path in the directory structure? Where do I register it?

**Comment by Diizzayy** [comment]
@Matman94 This requires creating a file in the `composables` directory and exporting your custom fetch instance.

What your looking for isn't currently possible, There isn't a way to globally override the default `useFetch` options, hence leveraging composables is the recommended approach.
